### ISSUE TITLE
across help page

### ISSUE THREAD

**Issue by ggrothendieck** [tidyverse/dplyr#5407]
The help page of across says "across() makes it easy to apply the same transformation to multiple columns, allowing you to use select() semantics inside in summarise() and mutate(). across() supersedes the family of "scoped variants" like summarise_at(), summarise_if(), and summarise_all()."

Is it true that it only works in summarize and mutate?   What about group_by?  other contexts?

**Comment by ggrothendieck** [comment]
With reference to https://stackoverflow.com/questions/62898859/how-can-i-use-dplyr-across-programmatically-on-no-variables/62901180#62901180 compute_mean1 gives an error but compute_mean2 and compute_mean3 give the expected answer.   compute_mean1 uses across and fails, coimpute_mean2 uses group_by_at and works while compute_mean3 is the same as coimpute_mean1 but also has the argument .add=TRUE and gives the expected answer. Surely this should work even if .add = FALSE.

```
compute_mean1 <- function(df, grouping) { # compute grouped mean with across()
  df %>% 
    group_by(across(all_of(grouping))) %>% 
    summarise(y = mean(y), .groups = "drop")
}

compute_mean2 <- function(df, grouping) { # compute grouped mean with group_by_at()
  df %>% 
    group_by_at(grouping) %>% 
    summarise(y = mean(y), .groups = "drop")
}

compute_mean3 <- function(df, grouping) { # compute grouped mean with across()
  df %>% 
    group_by(across(all_of(grouping)), .add = TRUE) %>%
    summarise(y = mean(y), .groups = "drop")
}

library(dplyr)

my_df <- tibble("x" = c("a", "a", "b", "b"), y = 1:4)

compute_mean1(my_df, NULL)

compute_mean2(my_df, NULL)

compute_mean3(my_df, NULL)
```

**Comment by iago-pssjd** [comment]
I was going to open an issue which could start by the same comment and end by the same question (other contexts?), so if you don't mind I will add here. I tried the next example:

```
> anscombe %>% 
      mutate(yn = case_when(any(across(matches("y\\d"), ~`>=`(., 9))) ~ 1, 
                            TRUE ~ 0))
Error: Problem with `mutate()` input `yn`.
x only defined on a data frame with all numeric variables
ℹ Input `yn` is `case_when(any(across(matches("y\\d"), ~. >= 9)) ~ 1, TRUE ~ 0)`.
Run `rlang::last_error()` to see where the error occurred.
```
where I wanted `yn` to be `1` if any of the variables `y1` to `y4` is greater than 9, or `0` otherwise (I use `case_when` instead of `if_else` since my goal example has more than 2 cases). I also tried with `any_of` instead of `any` and `vars` instead of `across`, but
```
x `any_of()` must be used within a *selecting* function.
```

Thank you!

**Comment by iago-pssjd** [comment]
I saw now my issue is addressed in issue #4770 .

**Comment by hadley** [comment]
You can use it in any "data-masking" function, of which `summarise()` and `mutate()` are the most common. I've attempted to clarify that in the docs.
